activity.js

```diff
@@ -25,12 +25,12 @@ class Activity {
     } else {
       timer.innerText = `${this.minutes}:${this.seconds}`;
     }
-  }
+  };
 
 
   markComplete() {
 
-  }
+  };
 
   saveToStorage() {
 
```

Comment:
thanks for your diligence on semicolons and other syntax/formatting. I definitely can use help with improving on that!!

Reply:
Same here. I've been sloppy with semi-colons. We'll need to make sure we do one final syntax review before submission.